x-pack/plugins/observability_solution/observability/server/routes/assistant/route.ts

```diff
@@ -12,9 +12,13 @@ import { createObservabilityServerRoute } from '../create_observability_server_r
 const getObservabilityAlertDetailsContextRoute = createObservabilityServerRoute({
   endpoint: 'GET /internal/observability/assistant/alert_details_contextual_insights',
   options: {
-    tags: [],
     access: 'internal',
   },
+  security: {
+    authz: {
+      requiredPrivileges: ['ai_assistant', 'rac'],
```

Comment:
Is `ai_assistant` a privilege? I'd expect something like `ai_assistant_contextual_details:read`

Reply:
Based on the below, `ai_assistant` is a privilege:
https://github.com/elastic/kibana/blob/c46d94ca8d5f783ace821064e1482d2fbdc6ab7c/x-pack/plugins/observability_solution/observability_ai_assistant/server/plugin.ts#L80

however, I'm a little confused now because adding `rac` didn't resolve the above error. 

@sorenlouv 
Do you know why these capabilities --> `ml:canGetJobs` or `ml:canGetDatafeeds` might be checked when fetching contextual insights?

Trying to figure out what's causing the error I posted above.

Reply:
This error is on the main branch too, before any of the authz changes to this endpoint. So I don't think it's related. However, I think we should get to the bottom of what's causing the error.